src/chart/bar/BarView.ts

```diff
@@ -980,6 +980,11 @@ function updateStyle(
         }
     );
 
+    const label = el.getTextContent();
+    if (!label) {
+        return;
+    }
+
     if (isPolar) {
         const position = itemModel.get(['label', 'position']);
         el.textConfig.inside = position === 'middle' ? true : null;
```

Comment:
It seems enough to put line 990 after the `setSectorTextRotation` function, in which has set the `textConfig`.
https://github.com/apache/echarts/blob/6bd27bf3a0454abd68098ece9966ef2943543f19/src/chart/bar/BarView.ts#L990

Reply:
Thanks, maybe we should put  before https://github.com/apache/echarts/blob/6bd27bf3a0454abd68098ece9966ef2943543f19/src/chart/bar/BarView.ts#L989-L996 for which set label rotation.